packages/office-ui-fabric-react/src/components/Persona/PersonaCoin/PersonaCoin.styles.ts

```diff
@@ -135,31 +135,31 @@ export const getStyles = (props: IPersonaCoinStyleProps): IPersonaCoinStyles =>
       },
 
       dimension < 32 && {
-        fontSize: FontSizes.mini
+        fontSize: fonts.xSmall.fontSize
```

Comment:
Is there a reason for going from mini -> xSmall? I know they're both 10px, but just curious.

Reply:
I just don’t like and don’t wanna use the `fonts.tiny` syntax 😄